Update README.md (#1)
adobe/storefront-product-listing-page: README.md

## Repo containing the Live Search PLP

> This repo is provided as a reference implementation only. While you’re expected to create your own customizations based on these assets, those customizations can’t be supported by Adobe.

Reviewer: ```suggestion
This repo is provided as a reference implementation only. While you are able to create your own customizations based on these assets, those customizations cannot be supported by Adobe.
```
